### ISSUE TITLE
🐛 Gateway/Warp is off on my domain 

### ISSUE THREAD

**Issue by razza-guhl** [cloudflare/cloudflared#796]
**Describe the bug**
The "require Gateway/WARP" policy in zero trust / Tunnel stopped working after a few weeks.

**To Reproduce**
1. Connect Warp or enrolled Warp (Zero Trust) client to Cloudflare
2. Request "/cdn-cgi/trace" endpoint of protected domain.
3. See results from output with connected Warp client:
```
fl=71f1400
h=xxx.de
ip=2a09:XXXX:XXXX:b0::XX:XXXX
ts=1667065497.001
visit_scheme=https
uag=Mozilla/5.0 (Linux; Android 12; M2012K11AG) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Mobile Safari/537.36
colo=FRA
sliver=none
http=http/2
loc=DE
tls=TLSv1.3
sni=plaintext
warp=off
gateway=off
kex=X25519
```

If it's an issue with Cloudflare Tunnel:
6. Tunnel ID : 00d23737-d4b8-4f65-86e1-4fb8f80440d6
7. cloudflared config: <docker>

**Expected behavior**

**Environment and versions**
 - OS: Windows 10, Android 12
 - Architecture: X86, ARM
 - Version: 2022.10.3

**Logs and errors**
-

**Additional context**
-

**Comment by abelinkinbio** [comment]
Hi @razza-guhl (and  @fliphess) - Thank you for reporting this issue. As a heads up, this is the GitHub repository for the `cloudflared` agent which is separate from the warp client. Unfortunately, there is no warp client repository since that client is not open sourced, but I have forwarded this issue to the right team internally. 

Additionally, I'd recommend reaching to Cloudflare Support and sharing the ticket number here so that I can continue looking into this report and gather more details.

**Comment by mekedron** [comment]
I was struggling with the same issues for a couple of hours.

And I want to leave here the solution for that, as it pops up in search engines quite high.

My problem was related to the warp client default profile. I switched Split Tunnels option into "Include IPs and domains" and I though it would be enough to put there 3 domain entries I want to have warp enabled for:
*.cloudflareaccess.com
example.com
*.example.com

When I tried to reach out my "subdomain.example.com" from my iPhone, it was saying that warp is disabled and I have no access. At the same time it was working fine on my MacBook.

After playing with it a couple of hours, I realized that it doesn't understand the asterisk in these settings.
So I replaced it with:
example.cloudflareaccess.com
example.com
subdomain.example.com

And hooray! I could successfully solve authorization problem.

This setting can be found here from main dashboard page.
Zero Trust -> Settings -> WARP Client -> Device Settings -> Profile Settings -> Default (or your custom) -> Split Tunnel.

Hope it might help someone!
